ast/src/main/java/com/graphicsfuzz/common/ast/visitors/AstBuilder.java

```diff
@@ -195,9 +198,124 @@ void addTopLevelDeclaration(Declaration decl) {
     topLevelDeclarations.add(decl);
   }
 
+  private static TranslationUnit fixUpArraySizes(TranslationUnit tu) {
```

Comment:
Please add a brief comment to explain the nature of the patching up, and that it is more convenient to do this after the AST has been built rather than during.